expr.go

```diff
@@ -132,6 +132,86 @@ func (neq NotEq) ToSql() (sql string, args []interface{}, err error) {
 	return Eq(neq).toSql(true)
 }
 
+// Lt is syntactic sugar for use with Where/Having/Set methods.
+// Ex:
+//     .Where(Lt{"id": 1})
+type Lt map[string]interface{}
+
+func (lt Lt) toSql(opposite, orEq bool) (sql string, args []interface{}, err error) {
+	var (
+		exprs   []string
+		opr     string = "<"
+		nullOpr string = "IS"
```

Comment:
I know it mirrors my implementation of Eq, but I think it's too confusing to treat nil/null this way for comparisons.